include/copc-lib/io/reader.hpp

```diff
@@ -37,6 +37,9 @@ class Reader : public BaseIO
     // Helper function to get all nodes from the root
     std::vector<Node> GetAllChildren() { return GetAllChildren(VoxelKey::BaseKey()); }
 
+    // Spatial querry helper functions
+    std::vector<Node> GetNodesWithinBox(const Box &box);
```

Comment:
Actually didn't we say we leave that on the user side? That function actually doesn't exist, I just forgot to remove the signature.

Reply:
Yeah.. it might be useful if the user just wants to know which Nodes are within the box, or get all the points in the box regardless of the node, not for our use case